src/current/cockroachcloud/costs.md

```diff
@@ -98,15 +102,15 @@ For Advanced clusters on AWS, each node will incur charges for the provisioned I
 
 For CockroachDB Basic clusters, storage is automatically provisioned on demand, eliminating the need for manual management. Costs are based on actual usage and adjust dynamically as data is added or removed, starting from zero.
 
-Storage is billed hourly, with [rates](https://www.cockroachlabs.com/pricing) depending on the cloud provider and the region where the cluster is hosted. Basic plan clusters charge for storage per GiB-hour. These charges are accumulated throughout the month to determine the total monthly storage charges.
+Storage is billed hourly, with [rates](https://www.cockroachlabs.com/pricing/new/) depending on the cloud provider and the region where the cluster is hosted. Basic plan clusters charge for storage per GiB-hour. These charges are accumulated throughout the month to determine the total monthly storage charges.
 
 </section>
 
 <section class="filter-content" markdown="1" data-scope="standard">
 
 For CockroachDB Standard clusters, storage is automatically provisioned on demand, eliminating the need for manual management. Costs are based on actual usage and adjust dynamically as data is added or removed, starting from zero.
 
-Storage is billed hourly, with [rates](https://www.cockroachlabs.com/pricing) specific to each cloud provider and the region where the cluster is hosted. CockroachDB Cloud measures storage in GiB-hour. These charges are accumulated throughout the month to determine the total monthly storage charges.
+Storage is billed hourly, with [rates](https://www.cockroachlabs.com/pricing/new/) specific to each cloud provider and the region where the cluster is hosted. CockroachDB Cloud measures storage in GiB-hour. These charges are accumulated throughout the month to determine the total monthly storage charges.
```

Comment:
Is GiB-hour defined somewhere? Total GiBs stored X hours that many GiBs have been stored?